examples/tsl2561_simpletest.py

```diff
@@ -43,7 +43,10 @@
 print("Integration time = {}".format(tsl.integration_time))
 print("Broadband = {}".format(broadband))
 print("Infrared = {}".format(infrared))
-print("Lux = {}".format(lux))
+if lux != None:
+    print("Lux = {}".format(lux))
+else:
+    print("Lux = NaN")
```

Comment:
How about one of the following? At a minimum, please consider changing the conditional syntax to be like these.
Only print when value exist:
```python
if lux is not None:
    print("Lux = {}".format(lux))    
```
**OR**, also print warning message when value is None:
```python
if lux is not None:
    print("Lux = {}".format(lux))  
else:
    print("None value returned. Sensor may be saturated.")
```

Reply:
it isn't only sensor saturation but also underrange.
2018-12-04T06:45:49-0500,NaN,0bb,0ir
2018-12-04T06:50:49-0500,NaN,0bb,0ir
2018-12-04T06:55:49-0500,0.5lux,1bb,0ir
2018-12-04T07:00:49-0500,0.2lux,2bb,1ir
.. is the output of my sensor logger from ten mins before to ten mins after sunrise.

point taken, though.